Sources_v3/StreamChatUI/MessageActionsPopup/ChatMessageActionsRouter.swift

```diff
@@ -0,0 +1,29 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+open class ChatMessageActionsRouter<ExtraData: ExtraDataTypes>: ChatRouter<ChatMessageActionsVC<ExtraData>> {
```

Comment:
Question: How granular the routers should be? Do we want each router to have just one function?